java/impl/aws/src/main/java/co/worklytics/psoxy/WebhookCollectionModeHandler.java

```diff
@@ -0,0 +1,174 @@
+package co.worklytics.psoxy;
+
+import co.worklytics.psoxy.aws.AwsContainer;
+import co.worklytics.psoxy.aws.SQSOutput;
+import co.worklytics.psoxy.aws.request.APIGatewayV2HTTPEventRequestAdapter;
+import co.worklytics.psoxy.gateway.HttpEventResponse;
+import co.worklytics.psoxy.gateway.ProcessedContent;
+import co.worklytics.psoxy.gateway.impl.BatchMergeHandler;
+import co.worklytics.psoxy.gateway.impl.InboundWebhookHandler;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPResponse;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+import co.worklytics.psoxy.aws.DaggerAwsContainer;
+
+/**
+ * AWS lambda entrypoint that can handle BOTH:
+ *    - webhook requests from API Gateway / Function URL invocations, AND
+ *    - SQS events (presumed to be batches of webhooks)
+ */
+public class WebhookCollectionModeHandler implements RequestStreamHandler {
```

Comment:
Rename then to `AWSWebhookCollectionModeHandler` or move it under `co.worklytics.psoxy.aws`